_episodes/05-memory.md

```diff
@@ -68,26 +68,32 @@ frequent formative assessment is important.
 >
 > Without looking at the list again, write down as many words from the list as
 > you can. How many did you remember? Write your answer in the Etherpad.
+> 
+> ## Accessibility Adaptation
+> The list above may also be read aloud, slowly and clearly, by the Trainer hosting the session. 
+> On your mark, ask learners to write down their answers where others cannot see them, then read the list 
+> again so they can check their work.
 {: .challenge}
 
-Most people will have found they only remember 5-7 words. Do not worry if you
-remembered fewer - many factors can affect short-term memory in different situations.
-Fortunately, there are techniques that we can use to improve our effective short-term memory.
-We will illustrate one of these techniques and then describe how this is applicable to our
-workshops.  
+Most people will have found they only remember 5-7 words. Those who 
+remember less may be experiencing distraction, fatigue, or (as we will learn shortly) "cognitive overload."  
+Those who remember more 
+are almost invariably deploying a *memory management strategy*.
 
 ## Strategies For Memory Management
 
-Our role as instructors, therefore, is to help our learners by not flooding their
-short term memory with too much information, partially by providing strategies
-and exercises to help them form the connections that will move concepts into
-long-term memory. What are some tools for doing this?
+Because short-term memory is limited, we can support learners by not flooding their
+short term memory with too many separate pieces of information. Does this mean we should 
+teach fewer concepts? Yes! However, this is not the only tool in our toolbox. 
+We can also assist by providing strategies
+and exercises to help them form the connections that will a) support them in holding more 
+things in short-term memory at once and b) begin to *consolidate* some concepts, moving them into
+long-term memory.
 
 ### Chunking
```

Comment:
The high jargon to text ratio here could really use a graphic with the terms together and some flowcharty magic. (it was already hard in the old version of a curriculum, even more now)